app/components/verkiezingen/bcsd-voorzitter-alert.js

```diff
@@ -0,0 +1,110 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { service } from '@ember/service';
+import { task, timeout } from 'ember-concurrency';
+import { findFirst } from 'frontend-lmb/utils/async-array-functions';
+import { queryRecord } from 'frontend-lmb/utils/query-record';
+import {
+  LID_VAST_BUREAU_CODE,
+  MANDAAT_LID_RMW_CODE,
+  VOORZITTER_BCSD_CODE,
+} from 'frontend-lmb/utils/well-known-uris';
+
+export default class VerkiezingenBcsdVoorzitterAlertComponent extends Component {
+  @service store;
+
+  @tracked errorMessage = '';
+
+  get bcsdBestuursorgaanInTijd() {
+    return this.args.bcsdBestuursorgaanInTijd;
+  }
+
+  get bestuursorganenInTijd() {
+    return this.args.bestuursorganenInTijd;
+  }
+
+  // Polling because otherwise changes in the installatievergadering data are not picked up
+  handleErrorMessage = task({ restartable: true }, async () => {
+    if (this.isDestroyed) {
+      return;
+    }
+
+    const voorzitter = await this.getVoorzitterBCSD();
```

Comment:
somehow the error message isn't shown if you add a new voorzitter that isn't in the required organs until you refresh.